src/core/sock/sockinfo_tcp.cpp

```diff
@@ -3187,6 +3187,14 @@ sockinfo_tcp *sockinfo_tcp::accept_clone()
     si->m_sock_state = TCP_SOCK_BOUND;
     si->setPassthrough(false);
 
+    // Inherit parent ring allocation logic
```

Comment:
lock is not replaced in case of RING_LOGIC_ISOLATE and delegated timers configuration. The best option is no inherit locks model during child socket object creation.

Reply:
I will do it separately not to block current progress